### DISCUSSION TITLE
In what time frame can we expecte the Release of V1.0.0?

### DISCUSSION THREAD

**Discussion by ThcDelux3** [robingenz/capacitor-firebase-authentication#60] [Q&A]
Looks like I great project. I would like to use this in a production app. How are your estimations to add the Facebook login support?

Best Regard Felix.

**Comment by robingenz** [comment]
Hi @ThcDelux3,
thank you for the question and your interest in this plugin.
The release of v1.0.0 will take some time, because I like to implement a few more features as currently marked until then.
Nevertheless, the current version is already very stable and can be used productively. 
I currently use it myself in a few apps (Android and iOS).
I will try to add support for Facebook this weekend.
I wanted to do that before version 1.0.0 anyway.

Best regards,
Robin

**Reply by ThcDelux3** [reply]
Thanks for the super-fast response. Sounds good will try the package out tomorrow. The roadmap looks great! Potential to become the best authentication package for Capacitor ;) keep up the great work. Props for the documentation.

**Comment by ThcDelux3** [comment]
Integrated the plugin successfully in my Vue Ionic Capacitor test app. Works like a charm 😊

Can you create a roadmap or improve the milestones (there are a bit confusing). I would like to help with some PR but I should know which features are the most important to you. So that V1.0.0 can become reality soon.

**Reply by robingenz** [reply]
I'm glad you like it. 
I just released a new version with support for Facebook Sign-In.
You can give it a try:
```
npm i @robingenz/capacitor-firebase-authentication@0.3.5
```
I will update the milestones and issues to make it a bit clearer.
